platform.txt

```diff
@@ -140,16 +140,20 @@ tools.blackmagicprobe.upload.speed=230400
 
 tools.blackmagicprobe.erase.params.verbose=
 tools.blackmagicprobe.erase.params.quiet=-q --batch-silent
-tools.blackmagicprobe.erase.pattern="{path}{cmd}" -quiet -ex "target extended-remote \\.\{serial.port}" -ex "monitor swdp_scan" -ex "attach 1" -ex "monitor erase mass" -ex "echo \nErase finished!\n" -ex "detach" -ex "quit"
+tools.blackmagicprobe.erase.pattern="{path}{cmd}" -quiet -ex "target extended-remote {serial.port}" -ex "monitor swdp_scan" -ex "attach 1" -ex "monitor erase mass" -ex "echo \nErase finished!\n" -ex "detach" -ex "quit"
+tools.blackmagicprobe.erase.pattern.windows="{path}{cmd}" -quiet -ex "target extended-remote \\.\{serial.port}" -ex "monitor swdp_scan" -ex "attach 1" -ex "monitor erase mass" -ex "echo \nErase finished!\n" -ex "detach" -ex "quit"
```

Comment:
Can we get a single pattern going if @micooke's idea from https://github.com/sandeepmistry/arduino-nRF5/issues/245#issuecomment-367192374 also works on Linux and Mac

It would be easy to maintain if both versions are the same.

Reply:
Sorry i meant to strike through that - if you read a bit further down the comment i found out that my idea didnt work. Hopefully @probonopd's update has the same affect.